src/ansys/fluent/core/services/field_data.py

```diff
@@ -222,13 +224,23 @@ class FieldUnavailable(RuntimeError):
     pass
 
 
-class SurfaceDataType(IntEnum):
+class SurfaceDataType(Enum):
     """Provides surface data types."""
 
-    Vertices = 1
-    FacesConnectivity = 2
-    FacesNormal = 3
-    FacesCentroid = 4
+    Vertices = "vertices"
+    FacesConnectivity = "faces"
+    FacesNormal = "face-normal"
+    FacesCentroid = "centroid"
+
+    @classmethod
+    def _missing_(cls, value: str):
```

Comment:
This method is same as `FluentEnum.__missing__`, so class can be derived from `FluentEnum` and the duplicate can be removed.

Reply:
Thank you @mkundu1, I agree. But here we will not need the missing method, so I was planning to remove it altogether only.
Thank you.